src/main/cpp/controllers/DrivetrainController.cpp

```diff
@@ -150,7 +150,7 @@ frc::TrajectoryConfig DrivetrainController::MakeTrajectoryConfig() {
 frc::TrajectoryConfig DrivetrainController::MakeTrajectoryConfig(
     units::meters_per_second_t startVelocity,
     units::meters_per_second_t endVelocity) {
-    frc::TrajectoryConfig config{kMaxV, kMaxA - 14.5_mps_sq};
+    frc::TrajectoryConfig config{kMaxV, kMaxA - 16.5_mps_sq};
```

Comment:
Should probably replace this with the result of `12/0.642 - 16.5`, which is `2.2_mps_sq`.